addons/sys_core/fnc_fastHashKeys.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: ACRE2Team
+ * Retrieves a list of keys inside an ACRE 2 hash or variable names defined in an object.
+ * This function can be accessed through the macro HASH_KEYS
+ *
+ * Arguments:
+ * 0: ACRE 2 hash <HASH> or object <OBJECT>
```

Comment:
Probably cleaner that way for future automated docs:
`* 0: ACRE 2 Hash <ARRAY>/<OBJECT>`